### PR TITLE
add media bundle standalone doc

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/media_standalone_display.rst [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+.. code-block:: yaml
+
+    doctrine_mongodb:
+        resolve_target_documents:
+                OpenOrchestra\ModelInterface\Model\TranslatedValueInterface: OpenOrchestra\MediaDemoBundle\Document\TranslatedValue
+                OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface: OpenOrchestra\MediaDemoBundle\Document\EmbedKeyword
+
+You have now the opportunity to query the database to get any of the media stored inside.
+
```
Comment (itkg-rboessel) [inline en/developer_guide/media_standalone_display.rst]: Maybe speak about database configuration

File: en/developer_guide/media_standalone_display.rst [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+Context
+-------
+
+To improve your website performances, a common way is to download the page datas from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url.
+
+An easy way to display would be to add a new virtual host to your existing Open Orchestra
+installation. This will allow you to make two parallel request to get the page (with the
+javascript and the css) and two parallel request to the media server (to get all the images).
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: plurals would be nice

File: en/developer_guide/media_standalone_display.rst [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+It is still possible to add ``MonologBundle`` for the logging.
+
+Installation with the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case that you will require to request the database to extract more
+information about the ``media`` object for instance. You will need to create two
+document : ``EmbedKeyword`` and ``TranslatedValue``.
+
+Those document are only present into the ``open-orchestra-model-bundle``.
```
Comment (itkg-rboessel) [inline en/developer_guide/media_standalone_display.rst]: Why you say that and after you say to create them inside our bundle. It's a little bit confusing

File: en/developer_guide/media_standalone_display.rst [hunk 4]
```diff
@@ -0,0 +1,183 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page datas from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: that was true for IE6 but now browsers have a limit around 10, no ?

File: en/developer_guide/media_standalone_display.rst [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    new OpenOrchestra\MediaModelBundle\OpenOrchestraMediaModelBundle(),
+
+It is still possible to add ``MonologBundle`` for the logging.
+
+Installation with the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case that you will require to request the database to extract more
+information about the ``media`` object for instance. You will need to create two
+document : ``EmbedKeyword`` and ``TranslatedValue``.
```
Comment (itkg-rboessel) [inline en/developer_guide/media_standalone_display.rst]: Maybe explain why you need them in one sentence.

File: en/developer_guide/media_standalone_display.rst [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+Open Orchestra Solution
+-----------------------
+
+A simple solution would be to install only the ``open-orchestra-media-bundle`` in a single
+``Symfony`` application.
+
+Basic installation
+~~~~~~~~~~~~~~~~~~
+
+In order to install all the symfony packages, we recommand you to use the
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: recommend

File: en/developer_guide/media_standalone_display.rst (hunk 5 above)
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: should be one sentence
documents

File: en/developer_guide/media_standalone_display.rst [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+            $this->value = $value;
+        }
+
+        public function getValue()
+        {
+            return $this->value;
+        }
+    }
+
+Once the document have been created inside your bundle, you should configure the interface
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: has been

File: en/developer_guide/media_standalone_display.rst [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+                OpenOrchestra\ModelInterface\Model\TranslatedValueInterface: OpenOrchestra\MediaDemoBundle\Document\TranslatedValue
+                OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface: OpenOrchestra\MediaDemoBundle\Document\EmbedKeyword
+
+You have now the opportunity to query the database to get any of the media stored inside.
+
+Ease your installation
+~~~~~~~~~~~~~~~~~~~~~~
+
+To facilitate this installation, we provide you with the ``open-orchestra-media-demo``
+repository which will show all those modifications.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: which implements ?

**Comment by nicolasThal** [comment]
@juchi @itkg-rboessel doc updated

File: en/developer_guide/media_standalone_display.rst [hunk 9]
```diff
@@ -0,0 +1,182 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page datas from
```
Comment (itkg-rboessel) [inline en/developer_guide/media_standalone_display.rst]: Data don't take s in plural form

File: en/developer_guide/media_standalone_display.rst [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url. Recent browser (such as ``Chrome`` or ``Firefox`` can perform
```
Comment (itkg-rboessel) [inline en/developer_guide/media_standalone_display.rst]: not clear when you speak about two request after. Why not ? 
In fact, browser can make a restraint number of parallel request to the same url.

**Comment by itkg-rboessel** [comment]
:+1:

File: en/developer_guide/media_standalone_display.rst [hunk 11]
```diff
@@ -0,0 +1,184 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page data from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url. Recent browser (such as ``Chrome`` or ``Firefox`` can perform
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: requests
the same domain ?
browsers

File: en/developer_guide/media_standalone_display.rst [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+up to 10 simultaneous requests). In the following documentation, we would just assume that
+browsers can make only two simultaneous request to the same domain.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: we will just assume
requests
Your assumption is not realistic, as you said above. I don't see the point of doing it.

File: en/developer_guide/media_standalone_display.rst [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+
+An easy way to display would be to add a new virtual host to your existing Open Orchestra
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: to display what ?

File: en/developer_guide/media_standalone_display.rst [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+installation. This will allow you to make two parallel requests to get the page (with the
+javascript and the css) and two parallel requests to the media server (to get all the images).
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: to avoid couting requests in your example (2 request to medias, 2 request to webserver, etc.) I would rephrase to something with the following idea "This will allow your browser to increase the number of parallel requests that can be done as several domain are available. It will then reduce the global latency of each page's download."

File: en/developer_guide/media_standalone_display.rst [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+
+Request scheme:
+
+.. image:: ../../images/two_parallel.gif
+.. image:: ../../images/four_parallel.gif
+
+This solution is not ideal for a main reason : you will need to boot the whole Open Orchestra
+kernel each time you will request an image.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst:23]: maybe just say "not ideal because you will need..." so it's faster to read

File: en/developer_guide/media_standalone_display.rst [hunk 16]
```diff
@@ -0,0 +1,184 @@
...
+Open Orchestra Solution
+-----------------------
+
+A simple solution would be to install only the ``open-orchestra-media-bundle`` in a single
+``Symfony`` application.
+
+Basic installation
+~~~~~~~~~~~~~~~~~~
+
+In order to install all the symfony packages, we recommend you to use the
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: Symfony

File: en/developer_guide/media_standalone_display.rst [hunk 17]
```diff
@@ -0,0 +1,184 @@
...
+
+.. code-block:: json
+
+    "require": {
+        "incenteev/composer-parameter-handler": "~2.0",
+        "open-orchestra/open-orchestra-base-bundle": "*",
+        "open-orchestra/open-orchestra-media-bundle": "*"
+    },
+
+Then activate all the required bundles in your ``AppKernel.php`` file:
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: activate -> enable ?

File: en/developer_guide/media_standalone_display.rst [hunk 18]
```diff
@@ -0,0 +1,184 @@
...
+    new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
+
+    new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
+    new Knp\Bundle\GaufretteBundle\KnpGaufretteBundle(),
+
+    new OpenOrchestra\BaseBundle\OpenOrchestraBaseBundle(),
+    new OpenOrchestra\MediaBundle\OpenOrchestraMediaBundle(),
+    new OpenOrchestra\MediaModelBundle\OpenOrchestraMediaModelBundle(),
+
+It is still possible to add ``MonologBundle`` for the logging.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: Good to know but why saying it ? Is there a reason one could think it's not possible ?

File: en/developer_guide/media_standalone_display.rst [hunk 19]
```diff
@@ -0,0 +1,184 @@
...
+
+Installation with the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case that you will require to request the database to extract more
+information about the ``media`` object for instance. You will need to create two
+documents : ``EmbedKeyword`` and ``TranslatedValue``.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: the first sentence ends abruptely

File: en/developer_guide/media_standalone_display.rst [hunk 20]
```diff
@@ -0,0 +1,184 @@
...
+resolution in the ``config.yml`` file:
+
+.. code-block:: yaml
+
+    doctrine_mongodb:
+        resolve_target_documents:
+                OpenOrchestra\ModelInterface\Model\TranslatedValueInterface: OpenOrchestra\MediaDemoBundle\Document\TranslatedValue
+                OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface: OpenOrchestra\MediaDemoBundle\Document\EmbedKeyword
+
+You have now the opportunity to query the database to get any of the media stored inside.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst:172]: I don't understand what is the purpose of these two documents. Shouldn't you give an explanation of what it lets you do or an example of usage ?

**Comment by nicolasThal** [comment]
@juchi doc updated

File: en/developer_guide/media_standalone_display.rst [hunk 21]
```diff
@@ -0,0 +1,186 @@
...
+
+.. code-block:: yaml
+
+    doctrine_mongodb:
+        resolve_target_documents:
+                OpenOrchestra\ModelInterface\Model\TranslatedValueInterface: OpenOrchestra\MediaDemoBundle\Document\TranslatedValue
+                OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface: OpenOrchestra\MediaDemoBundle\Document\EmbedKeyword
+
+Those documents should have been created because the ``Media`` document has some relations to
+some documents in an external bundle.
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst]: i would just move the explanation before the code, so it comes straight away.
And maybe say which "external bundle" requires them, as a user I wonder which one.

File: en/developer_guide/media_standalone_display.rst [hunk 22]
```diff
@@ -0,0 +1,186 @@
+Medias display
+==============
+
+Context
+-------
```
Comment (juchi) [inline en/developer_guide/media_standalone_display.rst:5]: I think I grasped the goal of this doc page. So I think you should start (before context) to say that there is a bundle that is available outside of the whole orchestra stack, standalone (apart for the BaseBundle I guess ?), etc. Some kind of intro so the reader understands that he can use this bundle in his sf project.

**Comment by nicolasThal** [comment]
@juchi doc updated

File: en/developer_guide/media_standalone_display.rst (hunk 22 above)
Comment (nicolasThal) [inline en/developer_guide/media_standalone_display.rst:5]: I have no clue on how i can introduce it... do you have one?

Comment (juchi) [inline en/developer_guide/media_standalone_display.rst:5]: Maybe smth like
"The section introduction the orchestraMediaBundle, a standalone bundle targeting media management/distribution/services/other? in a Symfony project. It can be used outside of the full orchestra stack so you can benefits from its features in any Symfony application."

Comment (nicolasThal) [inline en/developer_guide/media_standalone_display.rst:5]: I am not convinced.

Moreover a say that we can install the media-bundle as a standalone application : https://github.com/open-orchestra/open-orchestra-docs/pull/81/files#diff-4c0f5d5e8374269fce28c2f5d65581e5R28

Comment (juchi) [inline en/developer_guide/media_standalone_display.rst:5]: that was just an idea, i'll leave it up to you ;)
